src/frontends/tensorflow/src/graph_iterator_proto_txt.hpp

```diff
@@ -0,0 +1,52 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <fstream>
+
+#include "google/protobuf/io/zero_copy_stream_impl.h"
+#include "google/protobuf/text_format.h"
+#include "graph_iterator_proto.hpp"
+#include "openvino/frontend/exception.hpp"
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+
+class GraphIteratorProtoTxt : public GraphIteratorProto {
+public:
+    template <typename T>
+    GraphIteratorProtoTxt(const std::basic_string<T>& path) : GraphIteratorProto() {
+        std::ifstream pbtxt_stream(path, std::ios::in);
+        FRONT_END_GENERAL_CHECK(pbtxt_stream && pbtxt_stream.is_open(), "Model file does not exist");
+        auto input_stream = std::make_shared<::google::protobuf::io::IstreamInputStream>(&pbtxt_stream);
```

Comment:
Strange place. Few lines below you check an input (!input_stream - error). Check here is missing.

Reply:
Good catch, let me fix it in the separate PR. Don't thing it is to critical for now.

Reply:
Fixed here: https://github.com/openvinotoolkit/openvino/pull/16619